flopy/utils/postprocessing.py

```diff
@@ -123,7 +123,7 @@ def get_transmissivities(
     return T
 
 
-def get_water_table(heads, nodata, per_idx=None):
+def get_water_table(heads, nodata, valid_min=-1e-4, valid_max=3e4):
```

Comment:
`valid_min` and `valid_max` seem to be arbitrary values. Initial values of `None` might be better so filtering is only done if the user specifies values other than `None`

Reply:
The goal with these was to set a default range of valid head values that would result in nodata values (outside this range) getting automatically masked without any additional user input. But maybe there's a smarter way to do this. In the MODFLOW 6 I/O instructions, it says that *Cells that have been assigned an IDOMAIN value of zero or less are assigned a head value of 1.0 x 10^30. Cells that have converted to dry are assigned a dry value of −1.0 x 10^30.* Would it make sense to instead have default arguments for ``dry_cell_value=-1e30`` and ``inactive_cell_value=1e30`` (which would correspond to ``hdry`` and ``hnoflo`` in MODFLOW 2005 language). Are there situations where additional no data values would need to be input or would that cover it?